Makefile

```diff
@@ -58,10 +58,15 @@ clean: deps
 update:
 	vagrant box update
 
+
 # setting CONTIV_NODES=<number> while calling 'make demo' can be used to bring
 # up a cluster of <number> nodes. By default <number> = 1
+ifeq ($(VAGRANT_SUPPORT), 1)
```

Comment:
Jenkins builds are failing because VAGRANT_SUPPORT is undefined. Please see if there is better way to do this